docs/HowToSupportTizenRtProtectedBuild.md

```diff
@@ -0,0 +1,343 @@
+# Protected Build Porting Guide
+
+
+## Contents
+- [Introduction](#introduction)
+	- [Prerequisites](#prerequisites)
+	- [Overview of Protected Build](#overview-of-protected-build)
+- [Steps for suporting protected build in a new chipset](#steps-for-supporting-protected-build-in-a-new-chipset)
+	- [Creating a new configuration for Protected Build](#creating-a-new-configuration-for-protected-build)
+	- [Creating linker scripts](#creating-linker-scripts)
+	- [Userspace Initialization](#userspace-initialization)
+	- [MPU Initialization](#mpu-initialization)
+	- [Allocation of kernel and user heaps](#allocation-of-kernel-and-user-heaps)
+	- [Changes in Make files](#changes-in-make-files)
+	- [Changes in the download method](#changes-in-the-download-method)
+
+
+## Introduction
+This guide explains the steps required to support protected build on a new Chipset.
+
+### Prerequisites
+- TizenRT 3.0 flat build needs to be successfully ported and tested on the target CHIPSET board.
+- The generic code relating to TizenRT protected build should already be present in the codebase (Configuration variable: CONFIG_ BUILD_PROTECTED)
+- The developer should have information about the memory map and memory organisation of the board and partition information of the flash memory. These details can be found in either the chipset's data sheet, project configuration file, linker or download scripts or in the memory map mentioned as part of the chip's source code.
```

Comment:
1. How about `organization` instead of `organisation`? Is it more common?
2. What is difference between `the memory map` and `memory organisation of the board and partition information of the flash memory`? It looks the same.

Reply:
The sentence has been re-framed. Kindly review.